strapi/strapi
# fix: sticky navbar width

### What does it do?

fixes the sticky navbar behavior that was broken:
![CleanShot 2024-09-06 at 10 54 14@2x](https://github.com/user-attachments/assets/1ff8faf4-8ecd-4f69-97d1-f69c352d2622)

### How to test it?

Test the configure the view page, scroll, the navbar should appear in a sticky way with content inside, and should not appear over the secondary menu.

You can also test it in other pages that use the sticky navbar, like the settings
